roles/openshift_logging/tasks/delete_logging.yaml

```diff
@@ -1,5 +1,19 @@
 ---
 # delete the deployment objects that we had created
+- oc_obj:
+    state: list
+    kind: dc
+    all_namespaces: true
+    selector: "component=es,provider=openshift"
```

Comment:
what about ops?
I feel like we have something that is specific to logging and should return any/all objects (we could keep it to dcs though)

Reply:
Can we move this logic into `tasks/main` since we seem to be using it in both install and delete?